### ISSUE TITLE
TWRP 3.0.2-0 @ N7 (Tilapia)   |   Feature Request [Suggestion]

### ISSUE THREAD

**Issue by harayz** [TeamWin/Team-Win-Recovery-Project#706]
this is just a simple suggestion for restore backup option to have a button, drop down menu or text-box area where users can choose and customize the follow-up action after recovery is completed. For example, a drop-down menu or radio buttons to boot (recovery/system/bootloader), shutdown or files to flash once recovery is completed. Additionally, a text-box to add arguments or a simple script may allow users / developers to further customize the post-recovery operations by automating some of the recurring steps involved in testing and troubleshooting processes.
basically, this can prevent noobies like myself from tapping on the screen every few minutes to check on the progress made from recovery or backup  processes . however, if this option is already available, documented and discussed somewhere online, please ignore this 'issue' request and point me to the right place where i can find further information, relevant resources or detailed instruction to achieve just that. thank you and keep on TWRP-ing
